developerportal/apps/common/models.py

```diff
@@ -0,0 +1,19 @@
+from wagtail.core.models import PageManager, Page
+
+from .forms import BasePageForm
+
+
+class PublishedPageManager(PageManager):
+    def get_queryset(self):
+        # This method restricts the query to only live (non-draft) pages - it
+        # doesn't guard against private pages, however these aren't supported
+        # by the static site build.
+        return super().get_queryset().live()
```

Comment:
That’s right. Specifically, the reason is that adding `.public()` here breaks running migrations because it tries to look up the `wagtailcore_pagerevision` table before it’s been created (hence Travis CI failing earlier). I’ll clarify this in the code comment.